src/utils/FormattingUtils.ts

```diff
@@ -0,0 +1,165 @@
+import { ethers, BigNumber } from "ethers";
+import { createFormatFunction } from "@uma/common";
+import { DEFAULT_BLOCKCHAIN_EXPLORER_DOMAIN, PUBLIC_NETWORKS } from "../constants";
+
+export { createFormatFunction };
+
+export type BigNumberish = ethers.BigNumberish;
+export type BN = ethers.BigNumber;
+
+/**
+ * Parse a stringified number into a BigNumber.
+ * @param num The number to parse.
+ * @returns The parsed BigNumber.
+ */
+export function toWei(num: BigNumberish): BN {
+  return ethers.utils.parseEther(num.toString());
+}
+
+/**
+ * Parse a stringified number into a BigNumber with 9 decimal places.
+ * @param num The number to parse.
+ * @returns The parsed BigNumber.
+ */
+export function toGWei(num: BigNumberish): BN {
+  return ethers.utils.parseUnits(num.toString(), 9);
+}
+
+/**
+ * Converts a stringified number into a BigNumber. If the string version of the num contains a `.` then it is a number which needs to be parsed to a string int.
+ * @param num The number to parse.
+ * @returns The parsed BigNumber.
+ * @note This is a temporary function until we can backport support for decimal points to @across-protocol/sdk-v2.
+ */
+export const toBN = (num: BigNumberish): BN => {
+  // If the string version of the num contains a `.` then it is a number which needs to be parsed to a string int.
+  if (num.toString().includes(".")) {
+    return BigNumber.from(parseInt(num.toString()));
+  }
+  return BigNumber.from(num.toString());
+};
+
+export const formatFeePct = (relayerFeePct: BigNumber): string => {
+  // 1e18 = 100% so 1e16 = 1%.
+  return createFormatFunction(2, 4, false, 16)(toBN(relayerFeePct).toString());
+};
+
+/**
+ * Creates an etherscan link for a transaction or address on a given network.
+ * @param txHashOrAddress The transaction hash or address to link to.
+ * @param chainId The chainId to link to.
+ * @returns The etherscan link. This link will be formatted in markdown.
+ */
+export function etherscanLink(txHashOrAddress: string, chainId: number | string): string {
```

Comment:
While we're at it, thoughts on this?

```suggestion
export function blockExplorerLink(txHashOrAddress: string, chainId: number | string): string {
```

Reply:
I think we should change the function signature in a different PR.

Reply:
Resolved in https://github.com/across-protocol/sdk-v2/pull/378